src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,45 @@
+.navigation {
+  &__list {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    list-style-type: none;
+    align-items: flex-end;
+  }
+
+  &__item {
+    text-transform: uppercase;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+    display: block;
+    height: 60px;
+    line-height: 60px;
+    text-decoration: none;
+    color: $nav-link-color;
```

Comment:
It's good practice to define colors in variables for maintainability and reusability. Make sure that the `$nav-link-color` and `$nav-link-hover-color` variables are defined in `src/styles/utils/variables.scss` and that they follow the naming convention and color palette of the project.